src/main/resources/application.properties

```diff
@@ -5,9 +5,6 @@ spring.datasource.password=postgres
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.jpa.database=POSTGRESQL
 spring.jpa.show-sql=true
-#spring.jpa.hibernate.properties.hibernate.id.new_generator_mappings=true
-#spring.jpa.hibernate.properties.hibernate.cache.use_second_level_cache=true
-#spring.jpa.hibernate.properties.hibernate.cache.use_query_cache=false
-#spring.jpa.hibernate.properties.hibernate.generate_statistics=true
+liquibase.change-log=classpath:liquibase-changeLog.xml
```

Comment:
Change location of the "liquibase-changeLog.xml" file to "classpath:db/liquibase/liquibase-changeLog.xml".